brave-spark-framework-filter/src/main/java/com/github/kristofa/brave/spark/BraveSparkFilter.java

```diff
@@ -0,0 +1,77 @@
+package com.github.kristofa.brave.spark;
+
+import com.github.kristofa.brave.*;

```

Comment:
no wildcard imports.. also new files are supposed to use 2 space indent.

easiest way is to use square codestyle and reformat all your code

https://github.com/square/java-code-styles